ucm2/conf.d/macaudio/desktops/mono/HiFi.conf

```diff
@@ -0,0 +1,59 @@
+# Mono desktops
+
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Mux' Primary"
+		cset "name='Speaker Playback Mux' Secondary"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+Define.JackDevice "0"
+If.cs42l84 {
+	Condition {
+		Type ControlExists
+		Control "name='Jack ADC WNF Switch'"
```

Comment:
I would prefer some like
```
Define.JackCodec "cs42l84" 
```
in  `MacBook Pro J314.conf`
and
```
If.jackcodeccs42l84 {
	Condition {
		Type String
		String1 "cs42l84"
		String2 "${var:JackCodec}"
	}
	True.Include.hp4.File "/conf.d/macaudio/cs42l84/Headphones.conf"
}
```
here. same for cs42l83 and the other `Hifi.conf` files.